### ISSUE TITLE
Location modal not working on some pages

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#7346]
Let's deal with this on Monday! 😅 

![image](https://user-images.githubusercontent.com/24359/73109758-1be0e800-3ed2-11ea-8278-cae1462c3221.png)

On https://publiclab.org/wiki/unearthing-pvd the `Add a location` button doesn't seem to work, so I wonder if it's interacting somehow with some of our other mapping code?

I see this error, though, which could be involved:

```
Uncaught TypeError: element.find(...).tooltip is not a function
    at postProcessContent (xml.js:30)
    at setupWiki (xml.js:30)
    at unearthing-pvd:2108
    at unearthing-pvd:2110
```

**Comment by jywarren** [comment]
Hi @nstjean just checking in about this as well, i'd super appreciate some help on this if you have a moment. I noticed this seems to be the case on both the wiki page above, and also on my dashboard. I can open a separate issue for that; it seems also odd because I already have a location on my profile - let's check if there is conditional logic on that template and we'll break it out as a separate issue!

**Comment by jywarren** [comment]
Thanks!

**Comment by nstjean** [comment]
Hmmm. I'll take a look. There were two FTOs posted regarding the button, I'll check to make sure both were merged!

**Comment by jywarren** [comment]
Ah, ok, thanks!

**Comment by nstjean** [comment]
Ahh here it is. A PR was never submitted for this! https://github.com/publiclab/plots2/issues/7266 . This is only for the Dashboard, though. It should be working on Wiki pages. I'll keep looking.

**Comment by nstjean** [comment]
It looks like that console error is stemming from this code here:

https://github.com/publiclab/plots2/blob/aa3b8644e365080e264e73590529ff03bb63cb08/app/views/like/_like.html.erb#L157-L209

It's happening if you click anywhere on the page, not just the button.

**Comment by nstjean** [comment]
It does look like that error is interfering though. Here is the same issue:
https://stable.publiclab.org/wiki/inline-maps
Possibly it is the inline maps triggering an error there somehow.

**Comment by nstjean** [comment]
I think this is related: https://github.com/publiclab/plots2/issues/7355

**Comment by jywarren** [comment]
I'm now seeing this on /unearthing-pvd:

```
Access to XMLHttpRequest at 'https://iaspub.epa.gov/enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/PREF_LONGITUDE/BEGINNING/71/rows/0:300/JSON?_=1615315565558' from origin 'https://publiclab.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I believe that's actually a 404 error not CORS: the URL directly gives: `The requested URL /enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/PREF_LONGITUDE/BEGINNING/71/rows/0:300/JSON was not found.`

I'll open an issue based on https://github.com/publiclab/plots2/pull/8887 for the tile error i'm seeing and let's circle back here.

**Comment by jywarren** [comment]
Working now!
